tbc/views.py

```diff
@@ -1197,20 +1197,19 @@ def ConvertNotebook(request, notebook_path=None):
     notebook_name = notebook_name[0].split(".")[0]
     path = path.split("/")[0:-1]
     path = "/".join(path)+"/"
```

Comment:
I know that this is not part of your changes, but it seems odd, please use os.path.join and os.path.split
The variable ```path``` is used multiple times for changes to data, this can easily be integrated and done more elegantly